docs/changelog-fragments/578.packaging.rst

```diff
@@ -0,0 +1 @@
+Fixed issues that prevented RPM from building -- by :user:`pbrezina`.
```

Comment:
FWIW this note is not very informative to the end users. It'd need to be written in a different style.